t/op/tiearray.t

```diff
@@ -93,7 +93,9 @@ our @ISA = 'Implement';
 
 # simulate indices -2 .. 2
 my $offset = 2;
+no warnings 'once';
 $NegIndex::NEGATIVE_INDICES = 1;
```

Comment:
another way to avoid the warning :-)
`$NegIndex::NEGATIVE_INDICES = $NegIndex::NEGATIVE_INDICES = 1; # no warnings 'once'`

Reply:
> another way to avoid the warning :-)
> `$NegIndex::NEGATIVE_INDICES = $NegIndex::NEGATIVE_INDICES = 1; # no warnings 'once'`

Absent the comment, this would not be self-documenting.  In fact, it would be a puzzlement.  I'll go with putting the assignment into a block and scope the warning to the block.